flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamGraphHasherV2.java

```diff
@@ -253,12 +252,9 @@ private boolean generateNodeHash(
 
 		if (LOG.isDebugEnabled()) {
 			String udfClassName = "";
-			if (node.getOperatorFactory() instanceof SimpleOperatorFactory) {
-				SimpleOperatorFactory factory = (SimpleOperatorFactory) node.getOperatorFactory();
-				if (factory.getOperator() instanceof AbstractUdfStreamOperator) {
-					udfClassName = ((AbstractUdfStreamOperator<?, ?>) factory.getOperator())
+			if (node.getOperatorFactory() instanceof UdfStreamOperatorFactory) {
+					udfClassName = ((UdfStreamOperatorFactory) node.getOperatorFactory()).getOperator()
```

Comment:
Can not we add `getUserFunction()` or `getUserFunctionName()` method to the `UdfStreamOperatorFactory` interface?